litellm/proxy/spend_tracking/spend_tracking_utils.py

```diff
@@ -484,6 +484,7 @@ def _sanitize_request_body_for_spend_logs_payload(
     Truncates strings longer than 1000 characters and handles nested dictionaries.
     """
     from litellm.constants import LITELLM_TRUNCATED_PAYLOAD_FIELD
+
     MAX_STRING_LENGTH = 1000
```

Comment:
instead - can you just make `MAX_STRING_LENGTH` a constant in constants.py and control it through a .env 

that feels cleaner 

you can call it something like `MAX_STRING_LENGTH_PROMPT_IN_DB`

Reply:
Done and looking good!